### DISCUSSION TITLE
Must be superuser to use mount (UBUNTU 20.04, new installation)

### DISCUSSION THREAD

**Discussion by delovelady** [automatic-ripping-machine/automatic-ripping-machine#574] [Q&A]
I have worked through this installation on UBUNTU 20.04 and everything looks good on paper.

However, when I insert a disk, there is no indication that arm tries to run.  So I run this command in the terminal, and it results in this error:

arm@r2d2:/home/arm   09/09 15:38:28
> /usr/bin/python3 /opt/arm/arm/ripper/main.py -d sr0
mount: /mnt/dev/sr0: must be superuser to use mount.
umount: /dev/sr0: not mounted.

After that it seems to hang without activity.  Reviewing articles that Google found for me related to this on UBUNTU, I have made the following entry into /etc/sudoers

arm ALL=(ALL) NOPASSWD: /usr/bin/mount, /usr/bin/umount

But this does not seem to have corrected the problem.  Attributes for /dev/mnt/sr0  are:

> ls -dl /mnt/dev/sr0/                 
drwxrwxrwx 2 arm arm 4096 Sep  9 15:36 /mnt/dev/sr0/

(Yes, I changed ownership of that folder to arm:arm when I created it.  Changing it to root:root makes no difference (I tried).

Any suggestions/

**Comment by shitwolfymakes** [comment]
What groups is `arm` a member of?

**Reply by delovelady** [reply]
arm is a member of cdrom and a member of arm.
